src/main/java/frc/robot/commands/DriveUpTheRampCommand.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
@Jhony279 has been working on these two commands.  I think you've just duplicated his work.  

Take a look at his code.  If you'd like to suggest changes or if the two of you want to work on it together, maybe you should have a shared branch that you both commit to.